<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checkvaccinestatusscreen2</class>
 <widget class="QDialog" name="Checkvaccinestatusscreen2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Co-vix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="firstdosLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>02. Date Of First Dose of Covid Vaccine And Vaccine Type : </string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ecf5fa;
border-radius: 20px ;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>1011</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>701</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 2px solid #F47723;
border-radius: 1px ;
color: Black;
font-size: 14px;
letter-spacing: 4px;
 </string>
     </property>
     <property name="placeholderText">
      <string>Check Vaccine Status </string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>50</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;:/Co-vix Images/serchicon.png&quot;);
 background-repeat: no-repeat;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="checkButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK btn.png&quot;);
background-repeat: no-repeat;
border-style: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>500</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK VACCINE  STATUS LBL.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/Co-vix logo header.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="thirddoseLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>04. Date Of Third Dose of Covid Vaccine And Vaccine Type : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
background-image: url(&quot;:/Co-vix Images/BACK Btn.png&quot;);
border: 0px solid #555555;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>620</y>
     <width>102</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK CLEAR btn.png&quot;);
background-repeat: no-repeat;
border-style: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sendoseLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>03. Date Of Second Dose of Covid Vaccine And Vaccine Type : </string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>01. Name : </string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>360</y>
     <width>1261</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #F47723;
border-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #0A5894;
border-radius: 25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <zorder>frame_6</zorder>
  <zorder>firstdosLbl</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>thirddoseLbl</zorder>
  <zorder>backButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>sendoseLbl</zorder>
  <zorder>nameLbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
